src/Microsoft.ML.HalLearners/OlsLinearRegression.cs

```diff
@@ -706,7 +706,7 @@ public override void SaveSummary(TextWriter writer, RoleMappedSchema schema)
                 for (int i = 0; i < coeffs.Length; i++)
                 {
                     var name = names.GetItemOrDefault(i);
-                    writer.WriteLine(format, i, DvText.Identical(name, DvText.Empty) ? $"f{i}" : name.ToString(),
+                    writer.WriteLine(format, i, ReadOnlyMemoryUtils.Equals(name, String.Empty.AsMemory()) ? $"f{i}" : name.ToString(),
```

Comment:
>ReadOnlyMemoryUtils.Equals(name, String.Empty.AsMemory()) [](start = 48, length = 57)

Considering why this thing was useful in the first place (distinguishing missing from empty values), a simple `name.Length == 0` is a perfectly adequate replacement for this `ReadOnlyMemoryUtils.Equals(name, String.Empty.AsMemory())`.

Also do note that a `default(ReadOnlyChar<char>)` *is* the empty char span.